build.gradle

```diff
@@ -53,4 +58,7 @@ ext {
     androidXTestVersion = '1.1.3'
     mockitoVersion = '4.3.1'
     archCoreVersion = '2.1.0'
+
+    //kotlin
+    coreKtxVersion = '1.9.0'
```

Comment:
Same issue as here: https://github.com/salemove/android-sdk-widgets/pull/407#discussion_r1048223724

Reply:
so do we have possibility to fix it somehow? @DavDo @andrews-moc  @gugalo 
who could do this? perhaps @AndriiHorishniiMOC

Reply:
I can do it tomorrow,  if it is not possible for @AndriiHorishniiMOC

Reply:
@DavDo @dukhovnyi
I'm working on it.

Reply:
Done
Now Widgets SDK requires `targetSdkVersion` >= 31. The previous value was 29. It shouldn't be a problem for our integrators because now Google requires `targetSdkVersion` 31 or higher.